_posts/2021-01-04-nightly-builds-ncdns-certinject.md

```diff
@@ -0,0 +1,8 @@
+---
+layout: post
+title: "Nightly Builds of ncdns and certinject"
+author: Jeremy Rand
+tags: [News]
+---
+
+The [Beta Downloads page]({{ "/download/betas/" | relative_url }}) now includes Nightly builds of ncdns (both the plain binaries and the Windows installer) and certinject.  For the #reckless among you who watched my Grayhat 2020 presentation or my 3-part blog series on certinject and negative overrides, the Nightly builds of the ncdns Windows installer are a decent way to try out MITM-resistant system-wide TLS on Windows.  Be warned: there is a bug in the current Nightlies that could, under unusual circumstances, break your Windows certificate store, so for now you should only be trying this in a VM that you're willing to sacrifice to science.
```

Comment:
This bug is fixed in latest Nightlies (it was the "Hardcoded Namecoin Magic Tag" bug), so the warning can be removed from the post.